gnomad/utils/annotations.py

```diff
@@ -56,6 +56,8 @@ def pop_max_expr(
     :return: Popmax struct
     """
     _pops_to_exclude = hl.literal(pops_to_exclude)
+
+    # pylint: disable=invalid-unary-operand-type
```

Comment:
Is this needed because of the `~` below? Will we need to add this comment to other places where we use the `~`?

Reply:
Yes, Pylint incorrectly detects the `~` below. It doesn't look like every `~` results in a false positive. I didn't spend much time looking for a pattern.

Alternatively, we could disable this message in the project-level Pylint config.

Reply:
I figured that since it wasn't causing a false positive on *every* instance of `~`, it could still provide some value.